doc/central_scheduler.rst

```diff
@@ -78,20 +78,20 @@ The task history has the following pages:
 
     .. figure:: history.png
        :alt: Recent history screenshot
-* ``/history/by_id/:id``
+* ``/history/by_id/id``
```

Comment:
The colon refers to the `id` being a path parameter, we should revert it to having the `:`

Reply:
Did I mention plugging in the URI with that colon resolves to error 404?

Reply:
@tashrifbillah `:id` is used as a convention in Luigi doc to indicate this is a variable to be replaced, the same purpose as `{id}` that is more commonly used today.

Reply:
> :id is used as a convention in Luigi doc to indicate this is a variable to be replaced

Did you mention it somewhere in the doc? I guess the question is why would we use an unfamiliar convention that confuses new users?

Reply:
Well. This is a question that needs some archaeology work. ;)

I think the colon convention was not invented by Luigi and is used by other frameworks as well, such as express.js.

What would you suggest instead of colon? Would `{id}` for example be better in this case?

Reply:
> Would {id} for example be better in this case?

I think so.

Reply:
Cool. Would you like to revise this PR a bit? @tashrifbillah

Reply:
Sure, the places where I changed, are those the only places where I should effect the `:` vs `{ }` change?

Reply:
That seems to be the case. I didn't find anything else after a quick search.

Reply:
Hi @honnix , sorry about the delay. I revised the PR as you suggested. How does it look now?

Reply:
Ping @honnix ...